### ISSUE TITLE
Provide teams with a way to evaluate their model's performance over time

### ISSUE THREAD

**Issue by Bisaloo** [epiforecasts/covid19-forecast-hub-europe#786]
Following a discussion with @jbracher:

It would be interesting to provide teams with a way to evaluate the performance of their model(s) in each country for each week.
They would be able to check in which countries the models are performing well/poorly, whether it improves over time, etc.

Currently, it's not possible to access this information directly since:
- scores are not presented by date but aggregated by horizon
- the navigation makes it difficult to see the quality of a given model for different countries

We might argue that this is not our job to do this and that teams are able to do this individually but it might provides an extra self-interested incentive to submit forecasts ("we will automatically compute diagnostics for you so you can see how to improve your model").

**Comment by jbracher** [comment]
Totally agree that this information should be available in more fine-grained resolution. Would also make it easier for us to monitor performance and provide detailed feedback.

**Comment by sbfnk** [comment]
I agree - some sort of "scorecard" that can be provided to teams would be a fantastic addition.
